src/test-nunit-summary.exe/BriefHtmlOutputTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// ***********************************************************************
+// Copyright(c) 2016 Charlie Poole
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// ***********************************************************************
+
+using System;
+using System.IO;
+using NUnit.Framework;
+
+namespace NUnit.Extras.Tests
+{
+    public class BriefHtmlOutputTests : XmlTransformerTests
+    {
+        protected override string Input
+        {
+            get { return "MockAssemblyTestResult-2.6.4.xml"; }
+        }
+
+        protected override string Output
+        {
+            get { return "MockAssemblyBriefSummary-2.6.4.html"; }
+        }
+
+        protected override string Options
+        {
+            get { return "-brief -html"; }
+        }
+
+        [Test]
+        public void CheckReport()
+        {
+            string expectedReport = File.ReadAllText("BriefHtmlOutput_ExpectedReport.html");
+            Assert.That(Report, Is.EqualTo(expectedReport));
+        }
```

Comment:
I changed this test to use the Golden Master approach. FileAssert didn't work very well because it doesn't give good output when there is an error. Comparing the reports as strings gets you a pointer to the spot where the first difference appears.